images.CI/macos/anka/Anka.Helpers.psm1

```diff
@@ -14,15 +14,16 @@ function Push-AnkaTemplateToRegistry {
     )
 
     # if registry uuid doesn't match than delete an image in registry
-    $images = anka --machine-readable registry --registry-path $RegistryUrl list | ConvertFrom-Json | ForEach-Object body
+    $AnkaCaCrtPath="$HOME/.config/anka/certs/anka-ca-crt.pem"
+    $images = anka --machine-readable registry --cacert $AnkaCaCrtPath --registry-path $RegistryUrl list | ConvertFrom-Json | ForEach-Object body
     $images | Where-Object name -eq $TemplateName | ForEach-Object {
         $id = $_.uuid
         Show-StringWithFormat "Deleting '$TemplateName[$id]' VM and '$TagName' tag"
-        $uri = '{0}/registry/vm?id={1}' -f $RegistryUrl, $id
-        Invoke-WebRequest -Uri $uri -Method Delete | Out-Null
+        $curlCommand='curl -X DELETE -k "{0}/registry/vm?id={1}"' -f $RegistryUrl, $id
```

Comment:
lets go with `-s` for curl to suppress the output, better add `Write-Host` before the call to inform what this command is supposed to do

Reply:
There s already `Show-StringWithFormat` which is a function using `Write-Host`, we dont need another message!

Reply:
Successful build: https://dev.azure.com/mseng/AzDevNext.Deploy/_build/results?buildId=19768713&view=logs&j=80d86624-2c9a-5cad-9434-144b838e69fb&t=829dae9b-42f5-5e1c-cb9a-4dfa99b6fb63